pkg/mimirpb/compat.go

```diff
@@ -53,6 +53,13 @@ func ToWriteRequest(lbls []labels.Labels, samples []Sample, exemplars []*Exempla
 	return req
 }
 
+func ToWriteRequestEphemeral(lbls []labels.Labels, samples []Sample, exemplars []*Exemplar, metadata []*MetricMetadata, source WriteRequest_SourceEnum) *WriteRequest {
```

Comment:
I guess the reason why this is here, instead of in the test file where it's used, is because `ToWriteRequest()` is also here? 
If the reason for this being here isn't consistency with `ToWriteRequest()` then I think it should rather be in the test file where it's used, as there doesn't seem to be another caller.

Reply:
I agree, moving it to the test file.

Reply:
Done in https://github.com/grafana/mimir/pull/3922/commits/d9f55b01d2a35d1feefa2300321fb10150c5b239